website/src/content/docs/docs/libraries/http/reference/decorators.md

```diff
@@ -231,7 +231,6 @@ op read(@header accept: string): {
 op create(
   @header({
```

Comment:
```suggestion
  @header(#{
```

I believe this also needs to be updated

Reply:
actually this file is generated from the doc in `packages/http/lib/decorators.tsp` so will need to update that one first